src/test/java/org/mockbukkit/mockbukkit/block/state/DecoratedPotStateMockTest.java

```diff
@@ -116,13 +116,13 @@ void getSnapshot()
 	@Test
 	void notEqualNull()
 	{
-		assertNotEquals(pot, null);
+		assertNotEquals(null, pot);
```

Comment:
Order of operations is important here.
```suggestion
		// Checks if the pot is not equal to null according to the pot equals method, don't swap
		assertNotEquals(pot, null);
```
Might not be necessary for null, anyhow, JUnit uses the equals method of the first object supplied in the assertEquals